source/include/shadow.h

```diff
@@ -109,153 +109,195 @@ typedef enum ShadowStatus
  * @brief The common prefix of all Shadow MQTT topics
  *        from here https://docs.aws.amazon.com/iot/latest/developerguide/device-shadow-mqtt.html.
  */
-#define SHADOW_PREFIX                     "$aws/things/"
+#define SHADOW_PREFIX                        "$aws/things/"
 
 /**
  * @ingroup shadow_constants
  * @brief The length of #SHADOW_PREFIX.
  */
-#define SHADOW_PREFIX_LENGTH              ( ( uint16_t ) ( sizeof( SHADOW_PREFIX ) - 1U ) )
+#define SHADOW_PREFIX_LENGTH                 ( ( uint16_t ) ( sizeof( SHADOW_PREFIX ) - 1U ) )
 
 /**
  * @ingroup shadow_constants
  * @brief The root of all unnamed "Classic" Shadow MQTT topics
  *        from here https://docs.aws.amazon.com/iot/latest/developerguide/device-shadow-mqtt.html.
  */
-#define SHADOW_CLASSIC_ROOT               "/shadow"
+#define SHADOW_CLASSIC_ROOT                  "/shadow"
 
 /**
  * @ingroup shadow_constants
  * @brief The length of #SHADOW_CLASSIC_ROOT.
  */
-#define SHADOW_CLASSIC_ROOT_LENGTH        ( ( uint16_t ) ( sizeof( SHADOW_CLASSIC_ROOT ) - 1U ) )
+#define SHADOW_CLASSIC_ROOT_LENGTH           ( ( uint16_t ) ( sizeof( SHADOW_CLASSIC_ROOT ) - 1U ) )
 
 /**
  * @ingroup shadow_constants
  * @brief The common root of all named Shadow MQTT topics
  *        from here https://docs.aws.amazon.com/iot/latest/developerguide/device-shadow-mqtt.html.
  */
-#define SHADOW_NAMED_ROOT                 "/shadow/name/"
+#define SHADOW_NAMED_ROOT                    "/shadow/name/"
 
 /**
  * @ingroup shadow_constants
  * @brief The length of #SHADOW_NAMED_ROOT.
  */
-#define SHADOW_NAMED_ROOT_LENGTH          ( ( uint16_t ) ( sizeof( SHADOW_NAMED_ROOT ) - 1U ) )
+#define SHADOW_NAMED_ROOT_LENGTH             ( ( uint16_t ) ( sizeof( SHADOW_NAMED_ROOT ) - 1U ) )
 
 /**
  * @ingroup shadow_constants
- * @brief The string representing a Shadow "DELETE" operation in a Shadow MQTT topic.
+ * @brief The string representing a Shadow "UPDATE" operation in a Shadow MQTT topic.
  */
-#define SHADOW_OP_DELETE                  "/delete"
+#define SHADOW_OP_UPDATE                     "/update"
 
 /**
  * @ingroup shadow_constants
- * @brief The length of #SHADOW_OP_DELETE.
+ * @brief The string representing a Shadow "UPDATE ACCEPTED" operation in a Shadow MQTT topic.
```

Comment:
"Update accepted" is not a Shadow operation. Shadow operations are GET, UPDATE and DELETE. I think having a new OP to the accepted and rejected MACROs may be confusing.

Reply:
Un-done by new commit today.